src/app_config.cpp

```diff
@@ -177,12 +180,15 @@ ConfigOpt *opts[] =
 
 // Divert settings
   new ConfigOptDefenition<double>(divert_PV_ratio, 1.1, "divert_PV_ratio", "dpr"),
-  new ConfigOptDefenition<double>(divert_attack_smoothing_factor, 0.4, "divert_attack_smoothing_factor", "da"),
-  new ConfigOptDefenition<double>(divert_decay_smoothing_factor, 0.05, "divert_decay_smoothing_factor", "dd"),
-  new ConfigOptDefenition<uint32_t>(divert_min_charge_time, (10 * 60), "divert_min_charge_time", "dt"),
+  new ConfigOptDefenition<uint32_t>(divert_attack_smoothing_time, 20, "divert_attack_smoothing_time", "das"),
+  new ConfigOptDefenition<uint32_t>(divert_decay_smoothing_time, 200, "divert_decay_smoothing_time", "dds"),
```

Comment:
```suggestion
  new ConfigOptDefenition<uint32_t>(divert_decay_smoothing_time, 600, "divert_decay_smoothing_time", "dds"),
```
a value of 600 is much closer to the `master` defaults (at least for the data sets we have for the simulator) I would also change the `divert_PV_ratio` for the 'no waste` profile to 0.5, although I think we need to look at that value it looks to bedoing three different jobs...

Reply:
nowaste profile is actually supposed to be the same as default config and match old default behavior. Use 0.5 pv ratio as default looks a bit aggressive :)

But I can add more presettings then. Should we start with default, no waste, no import ?

Reply:
Here are the description I've put in UI for the "no waste" and "no import" presettings:
no waste: "No waste of produced energy. Slowly decrease charge rate, using grid to compensate, but increase faster when energy is going back."
no import: "Try to limit grid usage. Will slow down the charge rate quickly, but increase slower when energy is going back."

If we introduce a default one,  how would we describe it to differentiate it from no waste ?

Reply:
Here are the description I've put in UI for the "no waste" and "no import" presettings:
no waste: "No waste of produced energy. Slowly decrease charge rate, using grid to compensate, but increase faster when energy is going back."
no import: "Try to limit grid usage. Will slow down the charge rate quickly, but increase slower when energy is going back."

If we introduce a default one,  how would we describe it to differentiate it from no waste ?